js/binance.js

```diff
@@ -6684,6 +6684,109 @@ module.exports = class binance extends Exchange {
         return this.filterByArray (result, 'symbol', symbols, false);
     }
 
+    async fetchPosition (symbol, params = {}) {
```

Comment:
If this method is implemented, it should be implemented for `swap` and `future` as well

You might also want to add `fetchPositions`